understanding/21/non-text-contrast.html

```diff
@@ -388,6 +388,16 @@ <h4>Infographics</h4>
 				<p>An infographic can use text which meets the other criteria to minimise the number of graphical objects required for understanding. For example, using text with sufficient contrast to provide the values in a chart. A long description would also be sufficient because then the infograph is not relied upon for understanding.</p>
 
 				</section>
+				
+				<section>
+					<h4>Symbolic text characters</h4>
+					<p>When text characters are used as symbols – used for their visual appearance, rather than <q>expressing something in human language</q> – they fall under the definition of <a>non-text content</a>.</p>
```

Comment:
Turned this sentence around to make it more consistent. I'm assuming that since "non-text content" is a definition, the publishing script for WCAG will automagically turn this `<a>non-text content</a>` into the appropriate link to the `#dfn-non-text-content`